react/src/pages/SessionLauncherPage.tsx

```diff
@@ -979,6 +999,150 @@ const SessionLauncherPage = () => {
                         </Form.Item>
                       </Flex>
                     </Form.Item>
+                    {supportBatchTimeout ? (
+                      <Form.Item
+                        label={
+                          <Flex gap="xxs">
+                            <Typography.Text>
+                              {t('session.launcher.BatchJobTimeoutDuration')}
+                            </Typography.Text>
+                            <Tooltip
+                              title={t(
+                                'session.launcher.BatchJobTimeoutDurationDesc',
+                              )}
+                            >
+                              <QuestionCircleOutlined />
+                            </Tooltip>
+                          </Flex>
+                        }
+                        extra={
+                          <Form.Item
+                            noStyle
+                            shouldUpdate={(prev, next) =>
+                              prev.batch.timeout !== next.batch.timeout ||
+                              prev.batch.timeoutUnit !== next.batch.timeoutUnit
+                            }
```

Comment:
We can use `dependencies={[['batch', 'timeoutEnabled'], ['batch', 'timeoutUnit']]}` instead of `shouldUpdate`